beamer/tests/agent/test_rate_limiter.py

```diff
@@ -6,7 +6,7 @@
 import structlog.testing
 
 import beamer.agent
-from beamer.agent import Agent
+from beamer.agent.agent import Agent
```

Comment:
This looks funny/silly as hell, but oh well. :sweat_smile:

Reply:
Yes, though the same. Is there a better way?

Reply:
Nah, with the current setup, don't think so. But it's fine, just a bit funny to look at.